src/features/student/components/toggles/TraceGroupProductionToggle/TraceGroupProductionToggle.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+import { AvToggle, type AvToggleProps } from '@avenirs-esr/avenirs-dsav'
+import { useAttrs } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+interface TraceGroupProductionToggleProps extends Omit<AvToggleProps, 'description' | 'activeText' | 'inactiveText'> {
+  description?: string
+  activeText?: string
+  inactiveText?: string
+}
+
+const props = defineProps<TraceGroupProductionToggleProps>()
+
+const { t } = useI18n()
+const attr = useAttrs()
+
+const avToggleProps = computed(() => ({
+  ...attr,
+  ...props,
+  description: props.description ?? t('student.components.traceGroupProductionToggle.description'),
+  activeText: props.activeText ?? t('global.avToggle.activeText'),
+  inactiveText: props.inactiveText ?? t('global.avToggle.inactiveText')
```

Comment:
```suggestion
  description: t('student.components.traceGroupProductionToggle.description'),
  activeText: t('global.avToggle.activeText'),
  inactiveText: t('global.avToggle.inactiveText')
```
A priori tu n'utilises jamais ces propriétés dans ton code

Reply:
@p-beger si tu fais ça, on peut plus passer un custom activeText ou inactiveText et on peut pas initialiser une valeur par defaut dans defineProps vu qu'il faut `t('')`

Reply:
@elrhourha je sais mais là a priori tu n'as jamais eu besoin de passer des custom activeText ou inactiveText donc ça sert à rien d'exposer ces propriétés